sdcm/nemesis.py

```diff
@@ -251,20 +263,56 @@ def disrupt_restart_with_resharding(self):
         self.log.info('Set back murmur3_partitioner_ignore_msb_bits value to 12')
         self.target_node.restart_node_with_resharding()
 
-    def _destroy_data(self):
-        # Send the script used to corrupt the DB
-        break_scylla = get_data_dir_path('break_scylla.sh')
-        self.target_node.remoter.send_files(break_scylla,
-                                            "/tmp/break_scylla.sh")
+    @retrying(n=10, allowed_exceptions=(NoKeyspaceFound, NoFilesFoundToDestroy))
+    def _choose_file_for_destroy(self, ks_cfs):
+        file_for_destroy = ''
+
+        ks_cf_for_destroy = random.choice(ks_cfs)  # expected value as: 'keyspace1.standard1'
+
+        ks_cf_for_destroy = ks_cf_for_destroy.replace('.', '/')
+        files = self.target_node.remoter.run("sudo find /var/lib/scylla/data/%s-* -type f" % ks_cf_for_destroy)
+        if files.stderr:
+            raise NoFilesFoundToDestroy('Failed to get data files for destroy in {}. Error: {}'.format(ks_cf_for_destroy,
+                                                                                                       files.stderr))
+
+        for file in files.stdout.split():
+            if not file or '/' not in file:
+                continue
+            file_path_splitted = file.split('/')
+            # The file name like: /var/lib/scylla/data/scylla_bench/test-f60e4f30c98f11e98d46000000000002/mc-220-big-Data.db
+            # For corruption we need to remove all files that their names are started from "mc-220-"
+            file_name = '-'.join(file_path_splitted[-1].split('-')[:2]) + '-*'
+            file_for_destroy = '/'.join(file_path_splitted[:-1] + [file_name])
+            self.log.debug('Selected files for destroy: {}'.format(file_for_destroy))
```

Comment:
Will this select all files that it finds?
Do I understand it correctly?

Reply:
Yes, correct. All files that their name is started from, for example, "mc-220-"

Reply:
Ok, I see now the break after choosing files.
The old script used to choose 3 sets of files and with a minimum size.
Let's keep that behaviour or at least something similar that choose 3 big file (sets).